MiBench/automotive/bitcount/src/bitcnts.c

```diff
@@ -52,42 +49,49 @@ int main(int argc, char *argv[])
     "Non-recursive bit count by bytes (AR)",
     "Shift and count bits"
   };
-  if (argc<2) {
-    fprintf(stderr,"Usage: bitcnts <iterations>\n");
+  if (argc < 2) {
```

Comment:
please do not change the formatting of the original code. It makes code review more difficult.